sgeop/geometry.py

```diff
@@ -133,6 +138,8 @@ def voronoi_skeleton(
     if buffer is None:
         buffer = max_segment_length * 20
     if not poly:
+        if not isinstance(lines, gpd.GeoSeries):
+            lines = gpd.GeoSeries(lines)
```

Comment:
Why?

Reply:
In `L143`,  the `lines` object must have a `total_bounds` attribute. The [current docstrings](https://github.com/uscuni/sgeop/blob/cf1774d85e6eb3628d9acf0ad583c8f001349fc5/sgeop/geometry.py#L114) state that something array-like is acceptable to pass in. If that's the case, `lines` must be converted to (at least) a `GeoSeries`, no?